src/Symfony/Component/HttpFoundation/Session/Storage/Proxy/SessionHandlerProxy.php

```diff
@@ -22,7 +24,7 @@ public function __construct(\SessionHandlerInterface $handler)
     {
         $this->handler = $handler;
         $this->wrapper = $handler instanceof \SessionHandler;
-        $this->saveHandlerName = $this->wrapper ? \ini_get('session.save_handler') : 'user';
+        $this->saveHandlerName = $this->wrapper || ($handler instanceof StrictSessionHandler && $handler->isWrapper()) ? \ini_get('session.save_handler') : 'user';
```

Comment:
shouldn't the previous live be patched instead?
`$this->wrapper = $handler instanceof StrictSessionHandler ? $handler->isWrapper() : $handler instanceof \SessionHandler;`

Reply:
Apparently isWrapper is never used, we should deprecate it IMHO. (in a follow up)

Reply:
You can ignore my original comment on this line :)